### PR TITLE
Feature/email verification

### PR DESCRIPTION
**Pull request by hbarnardt**
Resolves issue #16 

Also the environment files have been updated with the SES details, and so many tests have been added

### CODE REVIEW

**Comment by hbarnardt** [comment]
If this PR is merge before PR #57  the environment files should be updated on #57

**Comment by avishkar58** [comment]
Lets just wait for @jaderabbit to review before merging.

File: api/app/users/api.py [hunk 1]
```diff
@@ -92,6 +95,10 @@ def post(self):
         except IntegrityError:
             return EMAIL_IN_USE
 
+        send_mail(recipient=user.email,
+                  subject='Password verification for IndabaDeepLearning portal',
```
Comment (jaderabbit) [inline api/app/users/api.py]: Let's change the subject to `Email verification for Deep Learning Indaba Portal`

Can we also use an environment variable/config for the URL?

**Comment by jaderabbit** [comment]
Also, I have merged the other PR. Would you refresh with dev?

**Comment by hbarnardt** [comment]
Will do

**Pull request merged**